3.15/localisation/en.json

```diff
@@ -0,0 +1,1771 @@
+{
+  "appName": "Optimize",
+  "appFullName": "Camunda Optimize",
+  "navigation": {
+    "analysis": "Analysis",
+    "events": "Event-based processes",
+    "logout": "Logout",
+    "logoutFailed": "Logout failed.",
+    "logoutSuccess": "You logged out successfully.",
+    "configLoadingError": "An error occurred while loading the UI configurations",
+    "documentation": "Documentation",
+    "help": "Help",
+    "profile": "Profile",
+    "appSwitcher": "App switcher",
+    "dashboards": "Dashboards",
+    "collections": "Collections",
+    "apps": {
```

Comment:
This is no longer part of the translation files in https://github.com/camunda/camunda-optimize/blob/master/optimize/backend/src/main/resources/localization/en.json.

Reply:
Correct, I updated the files now.